types/relay-runtime/lib/util/ReaderNode.d.ts

```diff
@@ -134,9 +134,10 @@ export interface ReaderRootArgument {
 }
 
 export interface ReaderRefetchMetadata {
-    readonly connection: ReaderPaginationMetadata | null | undefined;
+    readonly connection?: ReaderPaginationMetadata | null;
     readonly operation: string | ConcreteRequest;
     readonly fragmentPathInResult: ReadonlyArray<string>;
+    readonly identifierField?: string;
```

Comment:
@alloy @sibelius im not 100% clued up with Flow. 

This is the reference type:
https://github.com/facebook/relay/blob/7c67b4750592e469d499128108fe16afe2adaf51/packages/relay-runtime/util/ReaderNode.js#L66-L71

Does that mean both `connection` and `identifierField`, are optional, and nullable? Maybe I should also add `null` to `identifierField`??

Reply:
@maraisr 

Super i tested my use-case with `"readonly identifierField?: string;"`  and it is working fine... 

Connection from my prospective can get undefined or null but custom identifierField field only undefined... I don`t know in what scenario it can become null...

But super thx for fixing ;)

Reply:
Correct, Flow’s Maybe type also accepts `null`, but you already figured that out :)